apps/frontend/src/components/fap/Proposals/AssignFapMemberToProposalModal.tsx

```diff
@@ -69,6 +69,7 @@ const AssignFapMemberToProposalModal = ({
           title="Select reviewers"
           selection={true}
           data={members}
+          emailSearch={false}
```

Comment:
Is there a specific reason why we want to disable this emailSearch here?

Reply:
Is it the same for everybody or this is only STFC specific?

Reply:
Currently stfc specific

Reply:
If it is so we can make it maybe as an app setting and if we or someone else want to use it then we can choose.

Reply:
Sorry I'm a bit confused, you want to make the email search feature into a setting? The email search I am disabling here is only used by stfc at the moment.

Reply:
Ah OK that's fine then. Sorry for missunderstanding

Reply:
I think it is fine